app/assets/javascripts/components/student_dashboard/components/lesson_card.es6.jsx

```diff
@@ -0,0 +1,46 @@
+class LessonCard extends React.Component{
+
+
+  render() {
+    return (
+      <div className="lesson-card">
```

Comment:
`card-container` must know that it inherits from `lesson-card`, so it needs to be nested